lib/tictactoe/ai/minimax_ai.rb

```diff
@@ -10,15 +10,27 @@ class MinimaxAi
       DEFAULT_ALPHA = -100
       DEFAULT_BETA = 100
 
-      def calculate_next_move(board, marker, alpha = DEFAULT_ALPHA, beta = DEFAULT_BETA, depth = 6)
+      def calculate_next_move(board, marker)
+        # check 1 move deep for a winning or blocking move
+        # prefer this to a depth first search to improve speed
+        winning_move = next_winning_move(board, marker)
```

Comment:
Smells here:
* temp variables
* checking for truthy values to query whether a command succeeded (nil check, then return value)
* 3 logical branches here, 3 return paths
* return values of helper methods are inconsistent